vpdq/cpp/io/vpdqio.cpp

```diff
@@ -50,7 +51,7 @@ bool loadHashesFromFileOrDie(
       tmp.push_back(substr);
     }
 
-    if (tmp.size() != 3) {
+    if (tmp.size() != 4) {
```

Comment:
nit: (missed this in an old code review) something more descriptive than tmp, like `row` or `frame_values` might be better/clear